tests/egl/gles2_help.c

```diff
@@ -77,15 +77,26 @@ static const char *vertexShaderText_rgba =
   "   v_texcoord  = texcoord;\n"
   "}\n";
 
+static PFNGLEGLIMAGETARGETTEXTURE2DOESPROC imageTargetTexture2DProc = NULL;
+
+void imageTargetTexture2D(EGLenum target, EGLImageKHR image)
+{
+    if (!imageTargetTexture2DProc) {
+        imageTargetTexture2DProc =
+            (void *) eglGetProcAddress("glEGLImageTargetTexture2DOES");
+    }
+    imageTargetTexture2DProc(target, image);
+}
+
```

Comment:
this function called by every frame.And imageTargetTexture2DProc never change in run time. It's better we can init it in init section.